watertap/core/util/initialization.py

```diff
@@ -46,19 +46,29 @@ def check_solve(results, checkpoint=None, logger=_log, fail_flag=False):
     """
     if check_optimal_termination(results):
         if checkpoint is None:
-            logger.info(f"Solve successful.")
+            logger.info(
+                "The solver failed to converge to an optimal solution."
+                "This suggests that the user provided infeasible inputs or that the model "
+                "is poorly scaled, poorly initialized, or degenerate."
```

Comment:
The logger message should indicate a successful solve occurred but you're replacing it with a message indicating a failure to converge.